Cargo.toml

```diff
@@ -95,6 +96,7 @@ modern-full = [
     "extra_check",
     "functions",
     "hooks",
+    "preupdate_hook",
```

Comment:
Please don't touch `modern-full` because most of the time SQLITE_ENABLE_PREUPDATE_HOOK is not activated by default. So if you use `modern-full` and link against the SQLite3 library installed on your OS, the build will probably fail.

Reply:
Good to know, thanks! Fixed.